### ISSUE TITLE
[Partially Resolved] tcArpGen latest beta does not output anything on Arp, H1, H2, or H3 outs. Only Poly outs work (MacOS)

### ISSUE THREAD

**Issue by fractalgee** [Tonecarver/tcRackModules#9]
Seems that the non-poly outs from tcArpGen are broken on the Mac and don't output anything. The poly outs are fine. Rack 2.3.0. MacOS 13.4, latest beta you posted on community of tcArpGen.

**Comment by Tonecarver** [comment]
Ok, thanks. Which rules file do you have loaded? Do the outputs work with other rules files? What color is the LED under the "Valid" label?

**Comment by fractalgee** [comment]
Found the issue. I saw the rules file in the right click menu, however turns out that you save the absolute path to it. The patch came from k-chaffin. Checked again and the rules file had his user name in it. Given the rules file is in the tcArpGen plugiins folder it should only save the relative path methinks, else the patches aren't sharable. That would be the bug. Seems to work now that I refreshed it from my own plugins folder.

**Comment by Tonecarver** [comment]
Yes, agreed. tcArpgen needs to be smarter about how it loads shared patches.
